tests/tests/test_cells/test_cells.py

```diff
@@ -31,65 +51,12 @@ def test_pos_from_file_name(cubes_dir):
     assert natsorted(positions) == natsorted(positions_validate)
 
 
-def test_group_cells_by_z(xml_path):
-    z_planes_validate = [
-        1272,
-        1273,
-        1274,
-        1275,
-        1276,
-        1277,
-        1278,
-        1279,
-        1280,
-        1281,
-        1282,
-        1283,
-        1284,
-        1285,
-        1286,
-        1287,
-        1288,
-        1289,
-        1290,
-        1291,
-        1292,
-        1294,
-        1295,
-        1296,
-        1297,
-        1298,
-    ]
-
-    cell_numbers_in_groups_validate = [
-        1,
-        3,
-        7,
-        8,
-        3,
-        1,
-        4,
-        3,
-        1,
-        2,
-        2,
-        1,
-        1,
-        2,
-        5,
-        2,
-        2,
-        2,
-        3,
-        1,
-        1,
-        6,
-        1,
-        1,
-        1,
-        1,
-    ]
-
+def test_group_cells_by_z(
+    xml_path, z_planes_validate, cell_numbers_in_groups_validate
+):
+    """
+    Test that a list of cells can be grouped by z plane
```

Comment:
```suggestion
    Regression test that a list of cells can be grouped by z plane
```
Slightly clearer about what is tested?